apps/etterlatte-beregning/src/main/kotlin/beregning/Beregning.kt

```diff
@@ -17,7 +18,10 @@ data class BeregningsperiodeDAO(
     val soeskenFlokk: List<String>?,
     val grunnbelopMnd: Int,
     val grunnbelop: Int,
-    val grunnlagMetadata: Metadata
+    val grunnlagMetadata: Metadata,
+    val trygdetid: Int,
+    val regelResultat: ObjectNode? = null,
```

Comment:
Hvorfor ikke JsonNode?

Reply:
Kan ikke `JsonNode` være både et objekt, men også et array? Et `regelResultat` skal kun være et objekt.

Reply:
@aryani1

Reply:
Hvis det kun kan være ett objekt så høres `ObjectNode` riktig ut. `JsonNode` er vilken gyldig json som helst, d.v.s det kan være string, int, boolean, objekt, array, null etc.

Reply:
Gjelder det argumentet hvis det er et objekt som endrer seg over tid også @aryani1  ?

Reply:
Jeg gikk for `JsonNode` i denne runden. Legger inn en lapp i postkasse så kan vi ta en liten diskusjon på det på onsdag. Gjelder jo i så fall mange steder.